libcxx/test/support/test_macros.h

```diff
@@ -148,6 +148,12 @@
 # define TEST_IS_CONSTANT_EVALUATED false
 #endif
 
+#if TEST_STD_VER >= 26
+#  define TEST_STD_AT_LEAST_26_OR_RUNTIME_EVALUATED true
+#else
+#  define TEST_STD_AT_LEAST_26_OR_RUNTIME_EVALUATED (!TEST_IS_CONSTANT_EVALUATED)
+#endif
+
```

Comment:
This looks out of order, C++26, C++23, C++20. Typically we do increasing order of the version. Can you fix that and as drive-by for C++23?

Reply:
Agreed!